{{cookiecutter.repo_name}}/src/components/Footer.jsx

```diff
@@ -0,0 +1,29 @@
+import { Paper, Typography } from '@material-ui/core';
+import PropTypes from 'prop-types';
+
+import Link from '@material-ui/core/Link';
+
+const Footer = ({ className }) => {
+    return (
+        <Paper className={className} elevation={0}>
+            <Typography variant="body2" color="textSecondary" align="center">
+                {'Copyright © '}
+                <Link color="inherit" href="https://www.lightmatter.com/">
+                    Lightmatter
+                </Link>{' '}
+                {new Date().getFullYear()}
+                {'.'}
+            </Typography>
+        </Paper>
+    );
+};
+
+Footer.propTypes = {
+    className: PropTypes.string,
+};
+
+Footer.defaultProps = {
```

Comment:
Default props are now deprecated for functional components. We should default to default props for arguments instead across the board unless it's a class based component. https://github.com/reactjs/rfcs/blob/createlement-rfc/text/0000-create-element-changes.md#deprecate-defaultprops-on-function-components

Reply:
Sooo this isn't supported yet by eslint in a smart way. You can modify the rule to never validate props for functional components but that is not a good solution so maybe we leave it